### ISSUE TITLE
0.0.141 crashes on startup

### ISSUE THREAD

**Issue by CorruptComputer** [flathub/com.discordapp.DiscordCanary#41]
The most recent version that was released crashes when opening the app.

Terminal output:
```
[user@host ~]$ flatpak run com.discordapp.DiscordCanary 
2022/10/27 06:14:38 socat[3] E "/run/user/1000/app/com.discordapp.DiscordCanary/discord-ipc-0" exists
Disabling updates already done
[35:1027/061439.209296:FATAL:fork_and_spawn.cc(196)] posix_spawn: No such file or directory (2)
```

When running outside the terminal, just through the app icon in GNOME. The Problem Reporting app gives this reason for the crash: `DiscordCanary killed by SIGTRAP`

I am using Fedora 36, and already confirmed that the rest of my system is up to date.

**Comment by Etaash-mathamsetty** [comment]
the discord devs forgot to include the chrome crashpad handler in this version since they updated electron (finally), so there's literally nothing we can do to cure their stupidity or to easily fix the flatpak without them fixing their mistake.

**Comment by CorruptComputer** [comment]
Fixed by 0.0.142
